spec/requests/api/v1/targets/show_spec.rb

```diff
@@ -38,4 +39,24 @@
       expect(response).to have_http_status(:not_found)
     end
   end
+
+  context 'when the user is not authenticated' do
+    subject { post api_v1_targets_url, params: target_params, as: :json }
```

Comment:
Leave one empty line after the subject line.

Reply:
I think this should not be here. This is a POST request, and the describe of this test is ` 'GET /api/v1/Targets/:id'`